### ISSUE TITLE
Ability to catch Trash

### ISSUE THREAD

**Issue by edkotkas** [Zattri/smallFry#9]

**Comment by Zattri** [comment]
Collectibles unlock different bait types perhaps? You should also be able to fish without bait?

**Comment by edkotkas** [comment]
Yeah, you'd probably start off fishing and get your first bait, or some rare item which would get you better bait maybe
